[2025-07-09T05:33:57.148+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-09T05:33:57.164+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_airflare_data.extract manual__2025-07-09T05:33:55.214243+00:00 map_index=1 [queued]>
[2025-07-09T05:33:57.170+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_airflare_data.extract manual__2025-07-09T05:33:55.214243+00:00 map_index=1 [queued]>
[2025-07-09T05:33:57.170+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-09T05:33:57.180+0000] {taskinstance.py:2330} INFO - Executing <Mapped(_PythonDecoratedOperator): extract> on 2025-07-09 05:33:55.214243+00:00
[2025-07-09T05:33:57.188+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=4112) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-09T05:33:57.190+0000] {standard_task_runner.py:63} INFO - Started process 4116 to run task
[2025-07-09T05:33:57.189+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_airflare_data', 'extract', 'manual__2025-07-09T05:33:55.214243+00:00', '--job-id', '291', '--raw', '--subdir', 'DAGS_FOLDER/Untitled-1.py', '--cfg-path', '/tmp/tmpq9xle8se', '--map-index', '1']
[2025-07-09T05:33:57.191+0000] {standard_task_runner.py:91} INFO - Job 291: Subtask extract
[2025-07-09T05:33:57.243+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_airflare_data.extract manual__2025-07-09T05:33:55.214243+00:00 map_index=1 [running]> on host e6a32b72ede0
[2025-07-09T05:33:57.324+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_airflare_data' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2025-07-09T05:33:55.214243+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-09T05:33:55.214243+00:00'
[2025-07-09T05:33:57.327+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-09T05:33:57.327+0000] {Untitled-1.py:111} INFO - Начало извлечения данных для направления HKT-IKT
[2025-07-09T05:33:58.458+0000] {Untitled-1.py:26} INFO - Создание подключения к S3
[2025-07-09T05:33:58.459+0000] {Untitled-1.py:31} INFO - Подключение установлено
[2025-07-09T05:33:58.467+0000] {base.py:84} INFO - Using connection ID 'minio' for task execution.
[2025-07-09T05:33:58.468+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio', conn_type='aws') credentials retrieved from login and password.
[2025-07-09T05:33:58.799+0000] {Untitled-1.py:37} INFO - Загрузка данных в S3
[2025-07-09T05:33:58.946+0000] {Untitled-1.py:48} INFO - Загрузка завершена успешно
[2025-07-09T05:33:58.947+0000] {Untitled-1.py:124} INFO - Данные успешно сохранены во временное хранилище
[2025-07-09T05:33:58.960+0000] {Untitled-1.py:129} INFO - Данные успешно извлечены и переданы в Xcom
[2025-07-09T05:33:58.962+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-09T05:33:58.963+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-09T05:33:58.971+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_airflare_data, task_id=extract, run_id=manual__2025-07-09T05:33:55.214243+00:00, map_index=1, execution_date=20250709T053355, start_date=20250709T053357, end_date=20250709T053358
[2025-07-09T05:33:59.008+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-09T05:33:59.035+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-09T05:33:59.037+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
